report/report.go

```diff
@@ -167,11 +170,11 @@ func MakeReport() Report {
 			WithLabel("deployment", "deployments"),
 
 		ReplicaSet: MakeTopology().
-			WithShape(Heptagon).
+			WithShape(Pentagon).
```

Comment:
I'd make that a triangle since it's the most weird-looking shape and we are going to get rid of replicasets soon.